dist/metadata.json

```diff
@@ -1,18 +1,6 @@
 {
   "latest": "c56dfb8763aeae3de27b7ca0d4453f6c96d34d44",
   "versions": {
-    "4369a96e787b29c91bae5996c66054717020082e": {
```

Comment:
We should leave the previous versions as-is (At least the ones with PDFs) and special case them in the UI: like we can miss latex sources for instance.